applications/admin.py

```diff
@@ -167,6 +184,13 @@ def get_queryset(self, request):
             .prefetch_related("content_object")
         )
 
+    def get_readonly_fields(self, request, obj=None):
+        # Only allow the content_type and object_id to be set if a new object is being created. Otherwise, set those
+        # to readonly.
+        if obj and obj.id is not None:
```

Comment:
If we're only allowing setting these fields on objects which don't exist yet, should this condition be flipped?

Reply:
`obj and obj.id is not None` means that the object has an id already, meaning it isn't a new object being created, hence we want these fields _added_ to the readonly field list